components/Forms/Radio.tsx

```diff
@@ -2,6 +2,7 @@ import { InputHTMLAttributes } from 'react'
 import { TypedKeyAndText } from '../../i18n/api'
 import { Tooltip } from '../Tooltip/tooltip'
 import { ErrorLabel } from './validation/ErrorLabel'
+import { FormLabel } from '@dts-stn/service-canada-design-system'
```

Comment:
Added but not used?

Reply:
Must have forgotten that there. I was playing around with it but there were issues. I will remove this